merge/merge.go

```diff
@@ -88,11 +88,11 @@ func GitHubMerge(ctx context.Context, input Input, repoLimiter *time.Ticker, mer
 	reviews, _, err := client.PullRequests.ListReviews(ctx, input.Org, input.Repo, input.PRNumber, &github.ListOptions{})
 	if input.RequireReviewApproval {
 		if len(reviews) == 0 {
-			return Output{Success: false}, fmt.Errorf("PR awaiting review")
+			return Output{Success: false}, fmt.Errorf("PR awaiting review. Use --ignore-review-approval to override this check.")
 		}
 		for _, r := range reviews {
 			if r.GetState() != "APPROVED" {
-				return Output{Success: false}, fmt.Errorf("PR is not approved. Review state is %s", r.GetState())
+				return Output{Success: false}, fmt.Errorf("PR is not approved. Review state is %s. Use --ignore-review-approval to override this check.", r.GetState())
 			}
 		}
 	}
```

Comment:
relatedly, could also improve line 82 to say that

`--ignore-build-status      Ignore whether or not builds are passing`

lets you override

https://github.com/Clever/microplane/pull/50/files#diff-24cc3f380ea342a2744dcb5c93f4b2adR82

Reply:
👍